doc/cephfs/nfs.rst

```diff
@@ -79,3 +79,223 @@ Current limitations
 
 - Per running ganesha daemon, FSAL_CEPH can only export one Ceph file system
   although multiple directories in a Ceph file system may be exported.
+
+Exporting over NFS clusters deployed using rook
+===============================================
+
+In this tutorial we will be using minikube for setting up a single node test
+cluster. If you already have a kubernetes cluster deployed, please directly
+start with rook part.
+
+.. note:: Configuration of this tutorial should not be used in a a real
+          production cluster. For the purpose of simplification, the security
+          aspects of Ceph are overlooked in this setup.
+
+`Minikube <https://kubernetes.io/docs/setup/learning-environment/minikube/>`_ Setup
+-----------------------------------------------------------------------------------
+
+This setup is done on Fedora 30 and following package versions were used::
+
+        minikube v1.4.0, Kubernetes v1.16.0, Docker 18.09.9, git 2.21.0, kubernetes-client 1.15.2-1, docker-common 1.13.1-68 and qemu-kvm 3.1.1-2.
+
+Installation::
+
+        dnf install kubernetes-client docker-common git qemu-kvm
+        cd /tmp
+        curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube
+        curl -LO https://storage.googleapis.com/minikube/releases/latest/docker-machine-driver-kvm2 && chmod +x docker-machine-driver-kvm2
+        cp minikube docker-machine-driver-kvm2 /usr/local/bin
```

Comment:
$ cp minikube docker-machine-driver-kvm2 /usr/local/bin
cp: target ‘/usr/local/bin’ is not a directory

Reply:
Personally, I think putting minikube setup directions in this is overkill. Maybe instead we should just mention that you need a k8s cluster for this, and suggest minikube for folks that don't have one available?

Reply:
> $ cp minikube docker-machine-driver-kvm2 /usr/local/bin
> cp: target ‘/usr/local/bin’ is not a directory

It is a directory in fedora.

Reply:
> Personally, I think putting minikube setup directions in this is overkill. Maybe instead we should just mention that you need a k8s cluster for this, and suggest minikube for folks that don't have one available?

Agreed, I have removed minikube deployment section.